### ISSUE TITLE
PyPI 2016.02.23 release: plugins/Lists/plugin.py IndentationError on build

### ISSUE THREAD

**Issue by koobs** [ProgVal/Limnoria#1218]
While attempting to update the FreeBSD Limnoria port to the latest PyPI release (2016.02.23), the following error is observable during the build:

```
Traceback (most recent call last):
  File "/usr/local/bin/supybot-test", line 201, in <module>
    pluginModule = plugin.loadPluginModule(pluginName)
  File "/usr/local/lib/python2.7/site-packages/supybot/plugin.py", line 65, in loadPluginModule
    module = imp.load_module(name, *moduleInfo)
  File "./build/lib/supybot/plugins/Lists/__init__.py", line 53, in <module>
    from . import plugin
  File "./build/lib/supybot/plugins/Lists/plugin.py", line 78
    Class = Lists
    ^
```

The directory/file is definitely in the sdist, but looking at the repository, I cant seem to find this plugin listed either in `master` or the latest tag (2016.02.03).

It seems the repository has also not had a tag added for the 2016.02.23 release

**Comment by GLolol** [comment]
It looks like the master-2016-02-03 tag is a typo; it should probably be master-2016-02-**2**3 (released yesterday).

**Comment by ProgVal** [comment]
fixed in 2016.02.24, as discussed on IRC.
